vector/src/main/res/values/strings.xml

```diff
@@ -3037,6 +3037,8 @@
     <string name="live_location_sharing_notification_description">Location sharing is in progress</string>
     <string name="labs_enable_live_location">Enable Live Location Sharing</string>
     <string name="labs_enable_live_location_summary">Temporary implementation: locations persist in room history</string>
+    <string name="live_location_bottom_sheet_stop_sharing">Stop sharing</string>
```

Comment:
Should we have a dedicated string for the bottom sheet or could we reuse the "Stop" string used in other views to stop a live location share?